### ISSUE TITLE
Is this not available in dbt 1.6?

### ISSUE THREAD

**Issue by pei0804** [InfuseAI/piperider#854]
**Is your feature request related to a problem? Please describe.**

The new manifest format does not seem to be supported.
I am developing in the following environment.
Do you plan to support this in the future?

```console
❯ poetry run dbt --version
Core:
  - installed: 1.6.0
  - latest:    1.6.0 - Up to date!

Plugins:
  - snowflake: 1.6.0 - Up to date!
```

```console
❯ piperider compare-reports --base /tmp/piperider-base/run.json --target /tmp/piperider-target/run.json -o /tmp/piperider-comparison
────────────────────────────────────────────────────────────────────────────────── Comparison report ──────────────────────────────────────────────────────────────────────────────────
Selected reports:
  Base:   /tmp/piperider-base/run.json
  Target: /tmp/piperider-target/run.json
Warning: Runtime Error
  Expected a schema version of "https://schemas.getdbt.com/dbt/manifest/v9.json" in input file, but found "https://schemas.getdbt.com/dbt/manifest/v10.json". Are you running with a
different version of dbt?. Got problem to generate changeset.
```

**Comment by popcornylu** [comment]
We will support dbt 1.6 in the next release (ETA 8/24)
You can try the nightly version and see if it works

```
$ pip install piperider-nightly   
$ piperider version                                                                                                                                                                                        
0.32.0.20230815
GitCommit: bf26319bb87dd83859d623d2aba5ad95b774f335
```

**Comment by pei0804** [comment]
Change detection is working, but no summary report is generated.

Column additions are detected.
![image](https://github.com/InfuseAI/piperider/assets/9821370/f417ae6e-bdf2-4b97-9d77-90c58ddfea51)

It does not appear in the summary report.
![image](https://github.com/InfuseAI/piperider/assets/9821370/8f6e7d35-bc7c-46cc-b517-bf00b0a71ce2)

piperider version.

```console
piperider version
0.32.0.20230815
GitCommit: bf26319bb87dd83859d623d2aba5ad95b774f335
```

at main branch.
```console
PIPERIDER_SKIP_TARGET_CHECK=1 piperider run --dbt-target-path target/ -o /tmp/piperider-base
```

at future branch.
```console
PIPERIDER_SKIP_TARGET_CHECK=1 piperider run --dbt-target-path target/ -o /tmp/piperider-target
```

```console
piperider compare-reports --debug --base /tmp/piperider-base/run.json --target /tmp/piperider-target/run.json -o /tmp/piperider-comparison
────────────────────────────────────────────────────────────────────────────────── Comparison report ──────────────────────────────────────────────────────────────────────────────────
Selected reports:
  Base:   /tmp/piperider-base/run.json
  Target: /tmp/piperider-target/run.json
Warning: Runtime Error
  Expected a schema version of "https://schemas.getdbt.com/dbt/manifest/v9.json" in input file, but found "https://schemas.getdbt.com/dbt/manifest/v10.json". Are you running with a
different version of dbt?. Got problem to generate changeset.


Comparison report: /tmp/piperider-comparison/index.html
```

So, I changed the `dbt_schema_version` in the following file from `https://schemas.getdbt.com/dbt/manifest/v10.json` to `https://schemas.getdbt.com/dbt/ manifest/v9.json` and now I get a summary report.

````
Selected reports:
  Base: /tmp/piperider-base/run.json
  Target: /tmp/piperider-target/run.json
```

**Comment by popcornylu** [comment]
Thank you for your feedback, we will troubleshoot the problem you mentioned.

**Comment by popcornylu** [comment]
Hi, after investigation, i can reproduce your case only when
1. base report is `https://schemas.getdbt.com/dbt/ manifest/v9.json`
2. target report is `https://schemas.getdbt.com/dbt/ manifest/v10.json`
3. The environment to run piperider is `dbt-core=1.5.x`

To solve this, just also upgrade the dbt package in your environment you run the PipeRider


Here is my reproducing steps.

Run compare-reports in dbt 1.5.5, target report is generated in dbt 1.6.0 environment.
```
$ pip list | grep 'piperider\|dbt'
dbt-core                  1.5.5
dbt-duckdb                1.5.2
dbt-extractor             0.4.1
piperider-nightly         0.32.0.20230817

[notice] A new release of pip is available: 23.0.1 -> 23.2.1
[notice] To update, run: pip install --upgrade pip

$ piperider compare-reports --last
 Comparison report 
Selected reports:
  Base:   /private/tmp/jaffle_shop/.piperider/outputs/dev-20230818082235/run.json
  Target: /private/tmp/jaffle_shop/.piperider/outputs/dev-20230818082336/run.json
Warning: Runtime Error
  Expected a schema version of "https://schemas.getdbt.com/dbt/manifest/v9.json" in input file, but found "https://schemas.getdbt.com/dbt/manifest/v10.json". Are you running with a different version of dbt?. Got problem to generate changeset.

Comparison report: /private/tmp/jaffle_shop/.piperider/comparisons/latest/index.html
```

Upgrade to dbt 1.6
```
$ source ./dbt16/bin/activate
$ pip list | grep 'piperider\|dbt'                                                                                                                            dbt16  base  3.0.0
dbt-core                 1.6.0
dbt-duckdb               1.6.0
dbt-extractor            0.4.1
dbt-semantic-interfaces  0.2.0
piperider-nightly        0.32.0.20230817

[notice] A new release of pip is available: 23.0.1 -> 23.2.1
[notice] To update, run: pip install --upgrade pip

$ piperider compare-reports --last                                                                                                                          Comparison report
Selected reports:
  Base:   /private/tmp/jaffle_shop/.piperider/outputs/dev-20230818082235/run.json
  Target: /private/tmp/jaffle_shop/.piperider/outputs/dev-20230818082336/run.json

Impact Summary:
  Code Changes: added=0, removed=0, modified=0
  Resource Impact: potentially_impacted=0, assessed=0, skipped=0, impacted=0

Comparison report: /private/tmp/jaffle_shop/.piperider/comparisons/latest/index.html
```

**Comment by pei0804** [comment]
I think i can solve this problem.
Great thank you.
